app/modules/individuals/resources.py

```diff
@@ -51,24 +75,104 @@ def get(self, args):
 
     @api.login_required(oauth_scopes=['individuals:write'])
     @api.parameters(parameters.CreateIndividualParameters())
-    @api.response(schemas.DetailedIndividualSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def post(self, args):
-
         """
         Create a new instance of Individual.
         """
+        request_in = {}
+        try:
+            request_in_ = json.loads(request.data)
+            request_in.update(request_in_)
+        except Exception:
+            pass
+
+        if (
+            'encounters' not in request_in
+            or not isinstance(request_in['encounters'], list)
+            or len(request_in['encounters']) < 1
+        ):
+            _cleanup_post_and_abort(
+                None,
+                None,
+                'A new Individual must reference at least one encounter',
+            )
+
+        response = current_app.edm.request_passthrough(
+            'individual.data', 'post', {'data': request_in}, ''
+        )
+
+        response_data = None
+        result_data = None
+
+        try:
+            response_data = response.json()
+        except Exception:
+            pass
+        if response.ok and response_data is not None:
+            result_data = response_data.get('result', None)
+
+        if (
+            not response.ok
+            or not response_data.get('success', False)
+            or result_data is None
+        ):
+            log.warning('Individual.post failed')
+            passed_message = {'message': {'key': 'error'}}
+            if response_data is not None and 'message' in response_data:
+                passed_message = response_data['message']
+            abort(success=False, passed_message=passed_message, message='Error', code=400)
+
+        if 'encounters' in request_in and 'encounters' not in result_data:
+            _cleanup_post_and_abort(
```

Comment:
definitely a fault here and in all calls to this function. 
_cleanup_post_and_abort takes 3 parameters, db, guid and message and in all cases the guid is passed as the db, there is no guid in any call.

Reply:
I think I've taken care of this issue and your other comments. Please let me know if you would like to see any other changes.